tests/e2e/e2e_setup_test.go

```diff
@@ -843,8 +843,9 @@ func (s *IntegrationTestSuite) runPriceFeeder() {
 				"7171/tcp": {{HostIP: "", HostPort: "7171"}},
 			},
 			Env: []string{
-				"UMEE_E2E_UMEE_VAL_KEY_DIR=/root/.umee",
-				fmt.Sprintf("UMEE_E2E_UMEE_VAL_KEY_PASS=%s", keyringPassphrase),
+				"PRICE_FEEDER_DIR=/root/.umee",
```

Comment:
By the way @adamewozniak & @alexanderbez, would there be any consequences to running e2e tests on the same machine that an actual umee node had been run?

e.g. the `/root/.umee` folder having real stuff in it that shouldn't be read or overwritten.

If so it might be better to generate a temp directory during e2e, though that would require work outside the scope of this PR.